build/dependencies.yaml

```diff
@@ -77,13 +77,13 @@ dependencies:
       match: GOLANG_VERSION
 
   - name: "k8s.gcr.io/debian-base"
-    version: 2.1.0
+    version: 2.1.1
```

Comment:
this has to happen _after_ the image exists.

Reply:
This version should be incremented to trigger the build, but the `dependents` version cannot be updated until the image is promoted.